### PR TITLE
LPD-28304 Migrate pageEditorAccessibility.testcase

### PR DESCRIPTION
**Pull request by veroglez**
https://liferay.atlassian.net/browse/LPD-28304

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/test/playwright/tests/layout-content-page-editor-web/keyboard.spec.ts [hunk 1]
```diff
@@ -157,3 +157,24 @@ test('focus order is correct', async ({
 
 	await expect(generalTab).toBeFocused();
 });
+
+test('check that it cannot be accessed by keyboard in disabled areas', async ({
```
Comment (sandrodw3) [inline modules/test/playwright/tests/layout-content-page-editor-web/keyboard.spec.ts:161]: This one could be a jest test for `DisabledArea` component?

File: modules/test/playwright/tests/layout-content-page-editor-web/menuDisplayWidget.spec.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+	const widgetDefinition = getWidgetDefinition({
+		id: getRandomString(),
+		widgetConfig: {
+			displayStyle: 'ddmTemplate_NAVBAR-LINKS-FTL',
+		},
+		widgetName:
+			'com_liferay_site_navigation_menu_web_portlet_SiteNavigationMenuPortlet',
+	});
+
+	// Create three pages, one of them with Menu Display widget
```
Comment (sandrodw3) [inline modules/test/playwright/tests/layout-content-page-editor-web/menuDisplayWidget.spec.ts:39]: Why do we need to create the first two pages?

File: modules/test/playwright/tests/layout-content-page-editor-web/lockedPage.spec.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+/**
```
Comment (sandrodw3) [inline modules/test/playwright/tests/layout-content-page-editor-web/lockedPage.spec.ts:1]: I think this test would make more sense in `layout-admin-web`, what do you think?

File: modules/test/playwright/tests/layout-content-page-editor-web/rules.spec.ts [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+	await pageEditorPage.goto(layout, site.friendlyUrlPath);
+
+	await pageEditorPage.goToSidebarTab('Page Rules');
+
+	await page.getByRole('button', {name: 'New Rule'}).click();
+
+	// Fill out a condition and an action
+
+	await page.getByLabel('New Rule').press('Tab');
```
Comment (sandrodw3) [inline modules/test/playwright/tests/layout-content-page-editor-web/rules.spec.ts:54]: Could we take advantage and put all this logic in `addRuleCondition` and `addRuleAction` methods? They will probably be useful in the future 😄

**Review by sandrodw3 — COMMENTED** [review]
Awesome 😍 just some questions, let me know what you think!

File: modules/test/playwright/tests/layout-content-page-editor-web/keyboard.spec.ts [hunk 5]
```diff
@@ -178,3 +178,54 @@ test('check that it cannot be accessed by keyboard in disabled areas', async ({
...
+	site,
+}) => {
+	const layout = await apiHelpers.headlessDelivery.createSitePage({
+		siteId: site.id,
+		title: getRandomString(),
+	});
+
+	await pageEditorPage.goto(layout, site.friendlyUrlPath);
+
+	// Open the experience selector
```
[the remark this answers is not in the document]
Comment (veroglez) [reply modules/test/playwright/tests/layout-content-page-editor-web/keyboard.spec.ts:195]: Just what we want is to test by keyboard, so the method of opening it with a click is not worth it in this case.

File: modules/test/playwright/tests/layout-content-page-editor-web/rules.spec.ts (hunk 4 above)
Comment (veroglez) [reply modules/test/playwright/tests/layout-content-page-editor-web/rules.spec.ts:54]: Yes, I think it's a good idea!

File: modules/test/playwright/tests/layout-content-page-editor-web/menuDisplayWidget.spec.ts (hunk 2 above)
Comment (veroglez) [reply modules/test/playwright/tests/layout-content-page-editor-web/menuDisplayWidget.spec.ts:39]: It is to see how the template behaves when it has several elements and some of them is a child

File: modules/test/playwright/tests/layout-content-page-editor-web/lockedPage.spec.ts (hunk 3 above)
Comment (veroglez) [reply modules/test/playwright/tests/layout-content-page-editor-web/lockedPage.spec.ts:1]: Yes, in fact the module where the .jsp is located is in layout-admin-web, I don't know why I put it there XD

**Comment by veroglez** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:playwright

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:forward:force

**Pull request closed without merging**